tests/Command/FunctionalTest.php

```diff
@@ -92,7 +92,7 @@ public function getCommandTests()
             ->method('getRouteCollection')
             ->willReturn(new RouteCollection());
         yield 'controller' => array(
-            new MakeController($router),
+            new MakeController($router, sys_get_temp_dir()),
```

Comment:
you should be able to remove this if you start using `FileManager::fileExists()` I believe

Reply:
I afraid we don't have direct access to the `FileManager` from _makers_ right now. However, still injecting it the problem about the path will persist here, because this is a test provider function and `$this->targetDir` is null at this moment.

Reply:
Bah, that's annoying. That's not a good reason to write the code wrong though, so we need to think of some clever way to fix the tests. Probably we should make `FileManager` a 3rd argument to `testCommands` and make each provider pass in its own `FileManager` (we could add a nice `createFileManager()` method to make that easy, it would do all the `sys_get_temp_dir()` stuff)

Reply:
I left things as is for now, maybe resolve this in another PR?